src/accordion/stateless-accordion.tsx

```diff
@@ -23,9 +23,18 @@ function StatelessAccordion({
   return (
     <Root data-baseweb="accordion" {...rootProps}>
       {React.Children.map(children, (child, index) => {
-        const key = child.key || String(index);
-        return React.cloneElement(child, {
-          disabled: child.props.disabled || disabled,
+        let normalizedChild =
+          typeof child === 'object' ? (
```

Comment:
Will `ReactIs.isElement(child)` handle the type cast?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-is/index.d.ts#L27

Reply:
yes, that would allow to remove typecast.

updated to use it, also added check for isPortal - that looks like another allowed case here